planemo/commands/cmd_container_register.py

```diff
@@ -163,17 +165,21 @@ def handle_pull_request(self, ctx, name, target_filename, packages_str, tools_st
             pull_request(ctx, self.target_repository, message=message)
 
     def write_targets(self, ctx, target_filename, mulled_targets):
-        with open(target_filename, "w") as f:
-            target_strings = list()
-            for target in mulled_targets:
-                if target.version:
-                    target_str = "%s=%s" % (target.package_name, target.version)
-                else:
-                    target_str = target.package_name
-                target_strings.append(target_str)
-            contents = ",".join(target_strings)
-            f.write(contents)
-            ctx.vlog("Wrote requirements [%s] to file [%s]" % (contents, target_filename))
+        with open(target_filename, "a") as f:
```

Comment:
I guess if the file already exists we should not append, isn't it? Is `a` really needed here?

Reply:
It's a leftover from appending to hash.tsv